test/cli/test-dag.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+const repoPath = require('./index').repoPath
+const describeOnlineAndOffline = require('../utils/on-and-off')
+const ipfs = require('../utils/ipfs-exec')(repoPath)
+
+describe('dag', () => {
+  describeOnlineAndOffline(repoPath, () => {
+    it('get', () => {
```

Comment:
Should probably have a test-case as well when we try to resolve a hash that does not exists.